test/sample-test/run_sample_test.py

```diff
@@ -111,7 +112,9 @@ def run(self):
 
     # TODO(numerology): Special treatment for TFX::OSS sample
     if self._testname == 'parameterized_tfx_oss':
-      self._test_args['pipeline-root'] = os.path.join(self._test_args['output'], 'tfx_taxi_simple')
+      self._test_args['pipeline-root'] = os.path.join(
+          self._test_args['output'],
+          'tfx_taxi_simple_' + str(int(time.time())))
```

Comment:
You might also use the `kfp.dsl.RUN_ID_PLACEHOLDER`.

Reply:
Good idea. I think that's better

Reply:
@Ark-kun  Done.